documentation/installation.md

```diff
@@ -133,18 +133,18 @@ subjects:
 
 ### 4. Install Kyverno
 
-To install a specific version, change the image tag with git tag in `install.yaml`.
+To install a specific version, download `install.yaml` and then change the image tag.
```

Comment:
It'd be helpful if we link to the install.yaml by:
```[install.yaml(https://raw.githubusercontent.com/nirmata/kyverno/master/definitions/install.yaml) ```

Reply:
added the link